internal/sourcecode/scan.go

```diff
@@ -601,6 +602,42 @@ For detailed documentation, see https://fly.dev/docs/django/
 	return s, nil
 }
 
+// setup Laravel with a sqlite database
+func configureLaravel(sourceDir string) (*SourceInfo, error) {
+	// Laravel projects contain the `artisan` command, all php projects contain `composer.json` (in theory!)
+	if !checksPass(sourceDir, fileExists("artisan", "composer.json")) {
+		return nil, nil
+	}
+
+	var t []SourceFile
+	if checksPass(sourceDir, dirContains("composer.json", "laravel/octane")) {
+		t = templates("templates/laravel/octane")
+	} else {
+		t = templates("templates/laravel/standard")
+	}
+
```

Comment:
I think `dirContains()` also returns `true` if the directory contains either of these files.

Reply:
The `dirContains()` function (conveniently!) seems to be finding content within a file in the current directory (although perhaps there's an argument here for renaming the function).

<img width="761" alt="image" src="https://user-images.githubusercontent.com/467411/172695471-df2d0717-7f53-4003-8dd7-918aee6cd3a3.png">

So, for example, the `configureLucky` detector runs:

```go
checksPass(sourceDir, dirContains("shard.yml", "lucky"))
```

Which finds file `shard.yml` in the `sourceDir` and searches it for string `lucky`.

Reply:
Renaming this would be good, I agree.